test/fixtures/linter/autofix/ansi/026_l016_line_length_includes_earlier_fixes/after.sql

```diff
@@ -2,10 +2,10 @@ SELECT
     abs(round(foo_bar_report.metricx-xxx_yyy_report.metricx)) as col_c_rel_diff,
     abs(
         (
-            round(
-                foo_bar_report.metricx-xxx_yyy_report.metricx
-            )/foo_bar_report.metricx
-        )*100
+            round(foo_bar_report.metricx-xxx_yyy_report.metricx)
+            /foo_bar_report.metricx
+        )
+        *100
```

Comment:
Why was `*100` moved to a separate line, even though it fits?

Not necessarily a problem, just curious.